index.html

```diff
@@ -1,47 +1,160 @@
+        </p>
+      </div>
+      <div class="row">
+        <div class="column center-label s-4-12">
+          <img src="{{ '/images/advertising/happy-team.jpg' | prepend: site.baseurl }}" alt="" />
+          <h3>Working as a team</h3>
+          <p>
+            For a team to succeed at Student Robotics, it needs to work
+            together. For many participants, this is the first time they're in a
+            team setting. This practical experience is invaluable for their
+            future careers.
+          </p>
+        </div>
+        <div class="column center-label s-4-12">
+          <img src="{{ '/images/advertising/team-coding.jpg' | prepend: site.baseurl }}" alt="" />
+          <h3>Writing real code</h3>
+          <p>
+            Without some serious code, even the most masterfully built robots
+            can't do very much. With the help of our kit, teams will program
+            their robot in Python to compete in the competition.
+          </p>
+        </div>
+        <div class="column center-label s-4-12">
+          <img src="{{ '/images/advertising/glue-gun.jpg' | prepend: site.baseurl }}" alt="" />
+          <h3>Building a robot</h3>
+          <p>
+            There's nothing quite like building something and bringing it to
+            life. Using our kit, teams will build a fully autonomous robot. The
+            best teams pour their imagination into the design and the look of
+            their robots.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Testimonials</h2>
+      <div class="row vertically-centered-row">
+        <div class="column m-9-12">
+          <blockquote class="text-italic">
+            Student Robotics provided me with the invaluable experience of working on a engineering project with a team.
+            I learned not only good engineering principals but the importance of being able to clearly communicate my
+            thoughts and reasoning.
+            Competing in Student Robotics opened the door to me directly entering the world of work as a Software
+            Engineer after completing sixth form.
+          </blockquote>
+          <p class="text-heavy text-center">— James Seden Smith</p>
+          <h5 class="text-center">
+            Student Robotics 2015 and 2016<br>
+            Team CLY
+          </h5>
+        </div>
+        <div class="column m-3-12 horizontally-centered-column">
+          <img class="testomonial-avatar"
+            src="{{ '/images/advertising/headshots/james-seden-smith.jpg' | prepend: site.baseurl }}" />
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel panel-highlight panel-thin">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center top-spacing">Competitor feedback from the last two years</h2>
```

Comment:
Suggestion: Should this read more like a sentence, so it flows better with each of the below points?

```suggestion
      <h2 class="text-center top-spacing">Of competitors surveyed from the last 2 years...</h2>
```